share/functions/fish_git_prompt.fish

```diff
@@ -686,7 +695,7 @@ function __fish_git_prompt_validate_chars --description "fish_git_prompt helper,
     __fish_git_prompt_set_char __fish_git_prompt_char_dirtystate '*' '✚'
     __fish_git_prompt_set_char __fish_git_prompt_char_invalidstate '#' '✖'
     __fish_git_prompt_set_char __fish_git_prompt_char_stagedstate '+' '●'
-    __fish_git_prompt_set_char __fish_git_prompt_char_stashstate '$'
+    __fish_git_prompt_set_char __fish_git_prompt_char_stashstate '$' '⚑'
```

Comment:
I have an issue with these glyphs - for one I don't see why the non-informative version should have a worse glyph (if anything the worse versions should just be a fallback), and I also don't see what most of these are trying to communicate. See #5726.

But that's probably a more general thing.

I see you've taken the symbol from the bash git prompt, so that's acceptable for now.

Reply:
Yeah, while I agree I also feel like that's a somewhat orthogonal issue and should be resolved together with the others in that change.